src/signals/incident-management/containers/IncidentDetail/components/MapDetail/index.js

```diff
@@ -3,36 +3,23 @@ import PropTypes from 'prop-types';
 import Map from 'components/Map';
 
 import { locationType } from 'shared/types';
-
 import './style.scss';
 
-const MapDetail = ({
-  value, hideAttribution, hideZoomControls, useSmallMarker, zoom,
-}) => {
+const MapDetail = ({ value, mapOptions, ...otherProps }) => {
   const location = value && value.geometrie && value.geometrie.coordinates;
-  const latlng = location ? { latitude: location[1], longitude: location[0] } : null;
-  return (
-    <div className="map-detail">
-      {latlng
-        ? (
-          <Map
-            latlng={latlng}
-            hideAttribution={hideAttribution}
-            hideZoomControls={hideZoomControls}
-            useSmallMarker={useSmallMarker}
-            zoom={zoom}
-          />
-        ) : ''}
-    </div>
-  );
+  const lat = location && location[1];
+  const lng = location && location[0];
+  const options = {
+    ...mapOptions,
+    center: [lat, lng],
+  };
+
+  return (lat && lng) ? <Map lat={lat} lng={lng} mapOptions={options} {...otherProps} /> : null;
 };
 
 MapDetail.propTypes = {
   value: locationType.isRequired,
-  hideAttribution: PropTypes.bool,
-  hideZoomControls: PropTypes.bool,
-  useSmallMarker: PropTypes.bool,
-  zoom: PropTypes.string,
+  mapOptions: PropTypes.shape({}).isRequired /** leaflet options */,
```

Comment:
Why should this prop be required? The `MapDetail` component already has default options set.